x-pack/plugin/esql/qa/server/mixed-cluster/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/mixed/MixedClusterEsqlSpecIT.java

```diff
@@ -72,10 +72,6 @@ public MixedClusterEsqlSpecIT(
     protected void shouldSkipTest(String testName) throws IOException {
         super.shouldSkipTest(testName);
         assumeTrue("Test " + testName + " is skipped on " + bwcVersion, isEnabled(testName, instructions, bwcVersion));
-        assumeFalse(
```

Comment:
Should the similar be removed from [MultiClusterSpecIT](https://github.com/elastic/elasticsearch/blob/09d5e7f20f8e218861ec4e80456844cf47b80517/x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java#L115) class as well?

Reply:
:+1: